101-azurefirewall-premium/README.md

```diff
@@ -0,0 +1,11 @@
+# Create Firewall Premium (IDPS,TLS Inspection,Web Categories)
```

Comment:
see pipeline results for changes needed here

Reply:
@bmoore-msft are you referring to the Fairfax badge? Is that needed if we do not support fairfax?